docs/fundamentals/diagnostics/runtime-exception-events.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "exception runtime events"
+description: See .NET runtime events that collect diagnostic information specific to the exceptions and exception handling.
+ms.date: "09/27/2020"
+ms.topic: reference
+helpviewer_keywords:
+  - "exception events" [.NET Core]"
```

Comment:
Can you check all of these files for these bad `"` in the helpviewer_keywords area? There is an extra quote in most of them and it's throwing errors 😄 

```suggestion
  - "exception events [.NET Core]"
```